.buildkite/scripts/llvm-fpm/publish.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+source .buildkite/scripts/common.sh
+
+makefile=${1}
+patterns=${2}
+
+are_files_changed "$patterns"
```

Comment:
If this function is changed, this should also be updated

```suggestion
if ! are_files_changed "$patterns" ; then
    exit 0
fi
```